__tests__/actionCreators/settingsActions.test.js

```diff
@@ -0,0 +1,38 @@
+import settingsActions from '../../actions/settingsActions';
+import { UPDATE_USER_SETTINGS } from '../../constants/index';
+
+const mockDispatch = jest.fn();
+
+describe('settingsActions tests', () => {
```

Comment:
Small thing but make sure you add the afterEach code here that resets the mockDispatch function. It's int he workoutAction tests but missing here

Reply:
Also I think you need to mock updateUserSettings in this file as well since it's called from updateSettings.

Reply:
sir yes sir